lib/active_admin/reform/forms.rb

```diff
@@ -0,0 +1,37 @@
+require 'active_support/concern'
+require_relative 'resource_controller/data_access'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.